backend/src/graphql/resolver/AdminResolver.test.ts

```diff
@@ -1324,4 +1328,252 @@ describe('AdminResolver', () => {
       })
     })
   })
+
+  describe('transaction links list', () => {
+    const variables = {
+      userId: 1, // dummy, may be replaced
+      filters: null,
+      currentPage: 1,
+      pageSize: 5,
+    }
+
+    describe('unauthenticated', () => {
+      it('returns an error', async () => {
+        await expect(
+          query({
+            query: listTransactionLinksAdmin,
+            variables,
+          }),
+        ).resolves.toEqual(
+          expect.objectContaining({
+            errors: [new GraphQLError('401 Unauthorized')],
+          }),
+        )
+      })
+    })
+
+    describe('authenticated', () => {
+      describe('without admin rights', () => {
+        beforeAll(async () => {
+          user = await userFactory(testEnv, bibiBloxberg)
+          await query({
+            query: login,
+            variables: { email: 'bibi@bloxberg.de', password: 'Aa12345_' },
+          })
+        })
+
+        afterAll(async () => {
+          await cleanDB()
+          resetToken()
+        })
+
+        it('returns an error', async () => {
+          await expect(
+            query({
+              query: listTransactionLinksAdmin,
+              variables,
+            }),
+          ).resolves.toEqual(
+            expect.objectContaining({
+              errors: [new GraphQLError('401 Unauthorized')],
+            }),
+          )
+        })
+      })
+
+      describe('with admin rights', () => {
+        const expectNoDeletedOrRedeemed = expect.objectContaining({
+          data: {
+            listTransactionLinksAdmin: {
+              linkCount: 6,
+              linkList: expect.not.arrayContaining([
+                expect.objectContaining({
+                  memo: 'Leider wollte niemand meine Gradidos zum Neujahr haben :(',
+                  createdAt: expect.any(String),
+                }),
+                expect.objectContaining({
+                  memo: 'Da habe ich mich wohl etwas übernommen.',
+                  deletedAt: expect.any(String),
+                }),
+              ]),
+            },
+          },
+        })
+
+        beforeAll(async () => {
+          // admin 'peter@lustig.de' has to exists for 'creationFactory'
+          admin = await userFactory(testEnv, peterLustig)
+
+          user = await userFactory(testEnv, bibiBloxberg)
+          variables.userId = user.id
+          // bibi needs GDDs
+          const bibisCreation = creations.find((creation) => creation.email === 'bibi@bloxberg.de')
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          await creationFactory(testEnv, bibisCreation!)
```

Comment:
It's the other way around, because in the `creationFactory` `peter@lustig.de` gets logged in and does this job.
If I login `peter@lustig.de` here already then it doesn't work.

Took me some time to figure that out. 😅